bin/alluxio-common.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Utility functions for alluxio scripts 
+
+function get_ramdisk_array() {
+  local tier_path=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.path)
+  local medium_type=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.mediumtype)
+  local patharray
+  local mediumtypearray
+  local oldifs=$IFS
+  IFS=','
+  read -ra patharray <<< "$tier_path"
+  read -ra mediumtypearray <<< "$medium_type"
+  
+  RAMDISKARRAY=()
```

Comment:
I'm not a huge fan of storing the result in a variable the caller needs to know about, but returning arrays in bash is messy, so I'm ok with the simple solution.

Reply:
yeah.. exactly my thought as well